src/components/Explorer/ExplorerFormField.jsx

```diff
@@ -44,7 +44,7 @@ class ExplorerFormField extends React.Component {
     if (isDateField) {
       return (<span style={{ width: fieldWidth }}>
         <DatePicker
-          ref={(ref) => { this.datepicker = ref; return null; }}
+          ref={(ref) => { this.datepicker = ref; return this.datepicker; }}
```

Comment:
I actually prefer returning null here to indicate specifcally that all this function does is an assignment